Vagrantfile

```diff
@@ -168,12 +173,12 @@ Vagrant.configure(2) do |config|  # the literal "2" is required.
       end
     master_config.vm.network "public_network", bridge: interface_guesses, mac: "be0000" + bevy_mac
     master_config.vm.synced_folder ".", "/vagrant", :owner => "vagrant", :group => "staff", :mount_options => ["umask=0002"]
-    if settings.has_key?('application_roots')  # additional shares for optional applications directories
-      settings['application_roots'].each do |share|  # formatted real-path=share-path
-        s = share.split('=')
-        master_config.vm.synced_folder s[0], "/#{s[1]}", :owner => "vagrant", :group => "staff", :mount_options => ["umask=0002"]
-      end
-    end
+    #if settings.has_key?('application_roots')  # additional shares for optional applications directories
+    #  settings['application_roots'].each do |share|  # formatted real-path=share-path
+    #    s = share.split('=')
+    #    master_config.vm.synced_folder s[0], "/#{s[1]}", :owner => "vagrant", :group => "staff", :mount_options => ["umask=0002"]
+    #  end
+    #end
```

Comment:
I suggest deleting this or adding an explanatory comment about why someone might want to uncomment it.